roles/pool/src/lib/mod.rs

```diff
@@ -2,3 +2,105 @@ pub mod error;
 pub mod mining_pool;
 pub mod status;
 pub mod template_receiver;
+
+use async_channel::{bounded, unbounded};
+
+use mining_pool::{get_coinbase_output, Configuration, Pool};
+use template_receiver::TemplateRx;
+use tracing::{error, info, warn};
+
+use tokio::select;
+
+pub struct PoolSv2;
+
+impl PoolSv2 {
+    pub async fn start(config: Configuration) {
+        let (status_tx, status_rx) = unbounded();
+        let (s_new_t, r_new_t) = bounded(10);
```

Comment:
nit: naming conventions for channels should be consistent with the rest of the code base.